MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -15,18 +15,17 @@ internal static class Sdl
     public static int Minor;
     public static int Patch;
 
-    private static unsafe string GetString(IntPtr handle)
+    private static string GetString(IntPtr handle)
     {
         if (handle == IntPtr.Zero)
             return "";
 
-        var ptr = (byte*) handle;
-        while (*ptr != 0)
-            ptr++;
+        int len = 0;
+        while (Marshal.ReadByte(handle, len) != 0)
```

Comment:
@cra0zy - Yes please remove it.

We have enough problems with reliability and stability of the code base to not start changing things without good reasons.  There is nothing wrong with `unsafe` code.